src/modules/Dashboard/features/Managers/Popups/ManagersRemoveAccessPopup/ManagersRemoveAccessPopup.tsx

```diff
@@ -92,17 +103,17 @@ export const ManagersRemoveAccessPopup = ({
           appletIds: selectedApplets.map((item) => item.id),
           userId: user.id,
         });
-        onClose();
         break;
       default:
+        refetchManagers();
```

Comment:
I think it needs to re fetch managers even if user click close icon instead of ok

Reply:
fixed